projects/docs/docs/commands/graph.md

```diff
@@ -15,6 +15,10 @@ saying goes, "one image is worth a thousand words":
 The command will output the dependency graph as an image, in the `png` format.
 You can also change the format to `dot` (see [DOT](<https://en.wikipedia.org/wiki/DOT_(graph_description_language)>))
 to get the raw contents of the graph.
+:::caution
+You can also change the format to `json` to export the graph as a JSON file. The JSON schema is subject
+to change and may be breaking across major versions.
+:::
```

Comment:
Should be `minor` version? Otherwise I don't see the point of the caution as a major is breaking by definition

Reply:
Major doesn't necessarily mean breaking though. I'm not sure if we should break the schema in minor versions.

Reply:
> I'm not sure if we should break the schema in minor versions.

I think we should put out a warning this might break even in minor versions. This enables to iterate fast on this part of the codebase without scaring users by putting out too many major versions.

Reply:
Ok sounds good!